.travis.yml

```diff
@@ -1,7 +1,14 @@
 language: clojure
-install: mvn install --quiet -DskipTests=true
+install: mvn dependency:resolve
 script: mvn test
+cache:
+  directories:
+  - $HOME/.m2
 jdk:
   - openjdk7
   - oraclejdk8
   - oraclejdk9
+  - oraclejdk10
+matrix:
+  allow_failures:
```

Comment:
please, help me debug this https://travis-ci.org/razum2um/pomegranate/jobs/403479139
cannot reproduce it locally, I think a thread died there, but don't understand why